### ISSUE TITLE
Additional Button Actions

### ISSUE THREAD

**Issue by zakpucci** [nomis6432/MMM-flicio#1]
Not particularly an issue, but curious. I'm going to be trying this out this week, any opportunity to support additional button actions? For example double tap, triple tap, long tap?

**Comment by nomis6432** [comment]
Yes, that should be possible. I use [fliclib-linux-hci](https://github.com/50ButtonsEach/fliclib-linux-hci) and it seems like they [support a number of different click types](https://github.com/50ButtonsEach/fliclib-linux-hci/blob/master/clientlib/nodejs/fliclibNodeJs.js):
```js
ClickType: {
			ButtonDown: 0,
			ButtonUp: 1,
			ButtonClick: 2,
			ButtonSingleClick: 3,
			ButtonDoubleClick: 4,
			ButtonHold: 5
		},
```
I didn't need this when I implemented this so it would require some changes to the code. I believe you should listen for the `"ButtonSingleOrDoubleClickOrHold"` event and check the `ClickType` of that event to determine which action was performed. I currently don't have any flic buttons to test this but feel free to implement this yourself if you want to.

**Comment by zakpucci** [comment]
@nomis6432 Great, will look into this. Open to a PR on this if I decide to go this route?

**Comment by nomis6432** [comment]
A PR would definitely be welcome.

**Comment by zakpucci** [comment]
@nomis6432 PR ready to go, but unable to submit the PR - Seems like an access issue. If you want to add me as a contributor or whatever needs to be done, happy to push the PR.

**Comment by nomis6432** [comment]
Has been implemented in PR #2
